micro-rdk/src/native/entry.rs

```diff
@@ -84,6 +88,14 @@ pub async fn serve_web_inner(
         (cfg_response, robot)
     };
 
+    #[cfg(feature = "data")]
+    // TODO: Spawn data task here. May have to move the initialization below to the task itself
+    let _data_manager_svc = DataManager::<StaticMemoryDataStore>::from_robot_and_config(
```

Comment:
I think at some point we will want DataManager to work in terms of `dyn DataStore`, but given that we currently have only one implementation, I don't mind making the binding static.

Presumably, though, at some point the choice of `DataStore` would be 1) defaulted and 2) configurable from the config.

Reply:
agreed, I'll put in a todo (plus JIRA ticket) to this effect